index.html

```diff
@@ -2249,6 +2246,34 @@ <h3>
           available underneath an icon).
         </p>
       </section>
+      <section>
+        <h3 id="installation-sec">
+          Launching a web application
+        </h3>
+        <p>
+          The steps to <dfn>launch a web application</dfn> is given by the
+          following algorithm. The algorithm takes a
+          [=Document/processed manifest=] |manifest:processed manifest|,
+          an optional |target URL:URL| and returns a
+          [=service worker client/window client=].
+        </p>
+        <ol class="algorithm">
+          <li>If <var>url</var> is unset, set <var>url</var> to
+              |manifest|.[=manifest/start_url=].
+          </li>
+          <li>Let |client| be the result of
+              <a href="https://html.spec.whatwg.org/multipage/browsers.html#creating-a-new-top-level-browsing-context">
+              creating a new top-level browsing context</a>.
+          </li>
```

Comment:
```suggestion
          <li>Let |client| be the result of
              <a data-cite="html#creating-a-new-top-level-browsing-context">
              creating a new top-level browsing context</a>.
          </li>
```

Reply:
We should speak to the HTML folks about this... if we need "creating a new top-level browsing context" to be exported, we should get them to do that (and they can sanity check this).

Reply:
Sounds good. Filed: https://github.com/whatwg/html/issues/8449

Reply:
They are going to export "creating a new top-level traversable" soon that this can use: https://github.com/whatwg/html/issues/8449#issuecomment-1294581251

Reply:
Rebased onto the new html concepts. Note that main branch now has ReSpec errors: https://github.com/w3c/manifest/issues/1061